eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/ImpTest.hs

```diff
@@ -292,6 +293,30 @@ registerDRep = do
   dreps `shouldSatisfy` Map.member (KeyHashObj khDRep)
   pure khDRep
 
+setupDRepWithoutUTxO ::
+  forall era.
+  ( ConwayEraTxCert era
+  , ShelleyEraImp era
+  ) =>
+  ImpTestM
+    era
+    ( KeyHash 'DRepRole (EraCrypto era)
+    , KeyHash 'Staking (EraCrypto era)
+    )
+setupDRepWithoutUTxO = do
+  drepKH <- registerDRep
+  delegatorKH <- freshKeyHash
+  submitTxAnn_ "Delegate to DRep" $
+    mkBasicTx mkBasicTxBody
+      & bodyTxL . certsTxBodyL
+        .~ SSeq.fromList
+          [ mkRegDepositDelegTxCert @era
+              (KeyHashObj delegatorKH)
+              (DelegVote (DRepCredential $ KeyHashObj drepKH))
+              zero
+          ]
+  pure (drepKH, delegatorKH)
```

Comment:
Is it the case that having an extra txOut changes the rewards? I think when not explicitly set, the root will be the txOut, so it's not technically "withoutUtxo". 
The name of the function is a little confusing in my opinion, and also kinda specific.
But maybe because I'm missing the semantics of it - why do we actually need it like this, what's the consequences?  If possible, I would either try to find a better name (to reflect the intention) or if not maybe move it to the spec itself?

Reply:
Ok, after looking more at the PR, I understand  now the intention. 
Still I think the name is a little confusing, but  I have no better idea. Maybe you could add a comment explaining the difference between this and `setupSingleDRep`   (I suppose that being, that the Drep won't have no moneyz)

Reply:
Sure! And no worries! :sweat_smile:  Thanks!